database/sqltypes/sqltypes.go

```diff
@@ -70,6 +70,11 @@ func MakeString(b []byte) Value {
 	return Value{String{b, false}}
 }
 
+// MakeUtf8String makes a String value from a []byte.
+func MakeUtf8String(s string) Value {
```

Comment:
These make functions are unsafe.  I suggest removing them (assuming we don't use them in our code base)